### PR TITLE
Fixes #78 Remove Resizable hack form UxHtmlLabel

### LINKED ISSUE #78: Remove Resizable hack form UxHtmlLabel

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@Yuri05 We tried to use an RichTextEdit html for label but it did not work as expected. Instead, I am formatting the text to be either html (real html) or devexpress html that can be displayed in a label (the same text is also used as pure HTML in the InstallationValidator)

File: src/OSPSuite.Assets/UIConstants.cs [hunk 1]
```diff
@@ -241,31 +241,53 @@ public static string DoYouWantToDeleteDirectory(string newDirectoryName)
          return $"Do you want to delete the directory '{newDirectoryName}' and continue?";
       }
 
-      public static void AppendListItem(string listItem, StringBuilder sb)
+      public static void AppendListItem(StringBuilder sb, bool html, string listItem, int index)
```
Comment (Yuri05) [inline src/OSPSuite.Assets/UIConstants.cs]: where is `int index` used?

File: src/OSPSuite.Assets/UIConstants.cs [hunk 2]
```diff
@@ -241,31 +241,53 @@ public static string DoYouWantToDeleteDirectory(string newDirectoryName)
...
       {
-         sb.Append($"<li>{listItem}</li>");
+         if(html)
+            sb.Append($"<li>{listItem}</li>");
+         else
+            sb.AppendLine($"   {index}: {listItem}");
```
Comment (msevestre) [inline src/OSPSuite.Assets/UIConstants.cs]: here?

Comment (Yuri05) [inline src/OSPSuite.Assets/UIConstants.cs]: lol

**Pull request merged**